CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+<img width="94" height="71" src="logo.svg" title="Autoprefixer logo by Anton Lovchikov">
+
+# Contributing to Autoprefixer
+Please feel free to contribute to Autoprefixer by submitting Pull Requests to add new features, ehancements, bug fixes, etc. If you're not sure whether a PR should be created for something, [open an issue](https://github.com/postcssAautoprefixer/issues) to ask about it first. Check the specific section depending on the type of contribution you want to make.
```

Comment:
Hi, I found a little typo: `ehancements` -> `enhancements`. Also here is broken link `https://github.com/postcssAautoprefixer/issues`

Reply:
PR is not finished. Maybe you want to continue it. Just pull that branch to your fork and continue to add commits.

Reply:
Done! Fixed on `2eda913`

Reply:
Great!